.travis.yml

```diff
@@ -9,6 +9,7 @@ script:
 - parcel build src/index.html --public-url ./
 before_deploy:
 - cp robots-nofollow.txt dist/robots.txt
+- ln -s dist/index.html dist/404.html
```

Comment:
Are the source/target the right way around?

Reply:
I believe so. the way I remember it is 'real thing first' since 2nd param is optional.

Reply:
OK, just checking!